doc/api/http.md

```diff
@@ -1453,6 +1478,18 @@ added: v0.1.90
 
 Stops the server from accepting new connections. See [`net.Server.close()`][].
 
+### `server.expiredConnections()`
```

Comment:
I'm not clear what this provides. If a request is expired, that means Node is no longer tracking it, correct? Similarly, if a connection has expired it should have also been removed by Node, yes? So what utility does this method provide?

Reply:
You are 90% right. The thing is that the Node evicts expired connection every `http.Server.connectionsCheckingInterval` milliseconds (default is 30s), so it can definitely happen that connections are expired but not evicted yet. Hence this method to eventually allow the user to do other things (like log these connections before evictions or similar).